tests/test_tensorflow_bigquery.py

```diff
@@ -0,0 +1,10 @@
+import unittest
```

Comment:
Is that a new test you were trying to add? Looks like it is still just a copy and paste.

Reply:
added the comment on why this test is so simple